packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -73,7 +73,7 @@ Future<bool> launch(
         message: 'To use webview or safariVC, you need to pass'
             'in a web URL. This $urlString is not a web URL.');
   }
-  bool previousAutomaticSystemUiAdjustment;
+  bool previousAutomaticSystemUiAdjustment = true;
```

Comment:
do you know why `true` instead of `false`?  Could you add a comment?

Reply:
I just added a comment. I think it should be true because if it's false when the statusBarBrightness is set, then there will be a delay in computing the system UI.